### PR TITLE
Fixed incorrect page redirection to incorrect URL on clicking patient header data

### PR DESCRIPTION
**Pull request by themoonraker13**
Refer: https://talk.openmrs.org/t/page-reload-on-clicking-patient-header-data/8287

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java [hunk 1]
```diff
@@ -119,6 +119,13 @@ public void started() {
             Context.getAdministrationService().purgeGlobalProperty(defaultDashboard);
         }
 
+        // to replace '/coreapps/clinicianfacing/patient.page?patientId={{patientId}}&amp;app=pih.app.clinicianDashboard'
+        GlobalProperty dashboardUrl = Context.getAdministrationService().getGlobalPropertyObject(CoreAppsConstants.GP_DASHBOARD_URL);
+        if (dashboardUrl != null) {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: Doesn't this overwrite the user's custom value?

Comment (themoonraker13) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: Yes. My bad. This was to replace existing Global Property value that has been set by OpenMRS during installation earlier.

Comment (themoonraker13) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: Agree 👍. Liquibase changeset shall go into core platform code then.

Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: Not exactly. This particular one would go to the module's liquibase.xml :)

Comment (themoonraker13) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: I guess we do not have `liquibase.xml` in coreapps module.

Comment (themoonraker13) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: Let me add that and generate a new PR :)

**Pull request closed without merging**